core/translate/result_row.rs

```diff
@@ -162,39 +162,14 @@ pub fn emit_result_row_and_limit(
     Ok(())
 }
 
-pub fn emit_offset(
-    program: &mut ProgramBuilder,
-    plan: &SelectPlan,
-    jump_to: BranchOffset,
-    reg_offset: Option<usize>,
-    resolver: &Resolver,
-) {
-    let Some(offset_expr) = &plan.offset else {
+pub fn emit_offset(program: &mut ProgramBuilder, jump_to: BranchOffset, reg_offset: Option<usize>) {
+    let Some(reg_offset) = &reg_offset else {
         return;
     };
 
-    if let Some(val) = try_fold_expr_to_i64(offset_expr) {
```

Comment:
Did this cause the bug? If so, why? AFAICT `Expr::Variable` returns `None` from this function